gh-131020: Pylauncher does not correctly detect a BOM  when searching for the shebang (#131021)
python/cpython: Misc/NEWS.d/next/Windows/2025-03-09-19-57-35.gh-issue-131020._c87wf.rst

> pylauncher (py) does not correctly detect a BOM when searching for the
> shebang. Fix by Chris Eibl.

Reviewer: ```suggestion
:source:`pylauncher <PC/launcher2.c>` correctly detects a BOM when searching for the
shebang. Fix by Chris Eibl.
```

Maybe there is a better way to link, but this should be fine.

Author: Maybe we should keep `py` in there, because this is how user face it?
